bot/helpers.py

```diff
@@ -206,12 +216,13 @@ def get_validator(address) -> (dict, None):
         return node
     else:
         response = requests.get(VALIDATORS_ENDPOINT + "/" + address)
+
         if response.status_code != 200:
-            if not is_lcd_reachable():
+            if response.status_code == 500 and ('validator does not exist' in response.json().get('error', '')):
```

Comment:
So if the lcd is reachable, but the node doesn't exist, we still delete the node right?

Reply:
yes, then the API returns the information explicitly that the address doesnt exist